fetch/api/basic/request-upload.any.js

```diff
@@ -152,3 +152,31 @@ promise_test(async (test) => {
   const text = await resp.text();
   assert_equals(text, "ok. Request was sent 1 times. 1 connections were created.");
 }, "Fetch with POST with ReadableStream on 421 response should return the response and not retry.");
+
+
+promise_test(async (test) => {
+  const resp = await fetch(
+    "/fetch/connection-pool/resources/network-partition-key.py?"
+    + `status=425&uuid=${token()}&partition_id=${get_host_info().ORIGIN}`
+    + `&dispatch=check_partition&addcounter=true`,
+    {method: "POST", body: "foobar"});
+  assert_equals(resp.status, 425);
+  const text = await resp.text();
+  assert_equals(text, "ok. Request was sent 2 times. 2 connections were created.");
+}, "Fetch with POST with text body on 425 response should be retried once on new connection.");
```

Comment:
Chrome only retires if method is not GET or is GET?

Reply:
It doesn't make sense to test 425 response with POST method because it is for Early Data.
Anyway, Chrome retries once if method is both GET or POST.